### ISSUE TITLE
TAP producers should emit `ok`, not `okay`

### ISSUE THREAD

**Issue by DKroot** [jwovens/plTAP#8]
[TAP](https://testanything.org) is clearly using `ok` in the protocol. `okay` is causing glitches e.g. with the Jenkins TAP plug-in.

**Comment by jwovens** [comment]
> [TAP](https://testanything.org) is clearly using `ok` in the protocol. `okay` is causing glitches e.g. with the Jenkins TAP plug-in.

You're right.  As you can see I've not looked at this for a **long** time and don't know how that slipped through.  It has been that long that I didn't see your reply from a year ago.  Are still interested in the project? Would you care to collaborate on it?  It's not my day job so my development is limited and to be honest I don't do a lot with Oracle these days.

**Comment by DKroot** [comment]
Yes: I'm interested in collaboration. The timing is a bit bad though: my only Oracle project is ending tomorrow. But I can try to install a run some version of Oracle locally.
